Include/VersionStrings.h

```diff
@@ -1,4 +1,4 @@
 #define NIKRELEASE "v2.3.0dev"
-#define NIKLIBVERSION 24
-#define NIKLIBVERSIONSTR "24"
+#define NIKLIBVERSION 25
```

Comment:
The library version was changed from 23 to 24 when work on NiKom 2.3 started. So there is no need to change this now.